pkg/application/app.go

```diff
@@ -21,10 +15,16 @@ import (
 	"github.com/oam-dev/kubevela/pkg/appfile/template"
 	cmdutil "github.com/oam-dev/kubevela/pkg/commands/util"
 	"github.com/oam-dev/kubevela/pkg/oam"
-	"github.com/oam-dev/kubevela/pkg/utils/env"
-	"github.com/oam-dev/kubevela/pkg/utils/system"
+	"github.com/oam-dev/kubevela/pkg/storage"
+	"github.com/oam-dev/kubevela/pkg/storage/driver"
 )
 
+var store *storage.Storage
+
+func init() {
+	store = storage.NewStorage("")
```

Comment:
The `storage` variable initialization is not well placed in the `init()` function.

Reply:
yes， please declare it explicitly

Reply:
How should I get the user-specified storage type?

Reply:
We could provide multiple ways, use the default package variable is OK.

We could provide more ways like:

1. system environment 
2. system configfile
3. startup arguments